server/engine/telemetry.go

```diff
@@ -34,6 +35,21 @@ func BuildSegmentPropertiesMap(db *gorm.DB) analytics.Properties {
 	return propertiesMap
 }
 
+func GetMetricFromMainOutputs(db *gorm.DB) {
```

Comment:
Functions starting with `Get...` are known as getters and they should return something. In this case we are storing metrics so we should name it something like `StoreMetricsFromMainOutputs`.

Reply:
Changed it to `StoreMetricsFromMainOutputs`